src/Evolution/Systems/GrMhd/ValenciaDivClean/NewmanHamlin.cpp

```diff
@@ -188,6 +188,7 @@ boost::optional<PrimitiveRecoveryData> NewmanHamlin::apply(
                                PrimitiveRecoveryData>                          \
   grmhd::ValenciaDivClean::PrimitiveRecoverySchemes::NewmanHamlin::apply<      \
       THERMODIM(data)>(                                                        \
+      const double /*initial_guess_pressure*/,                                 \
```

Comment:
why is the name of the variable commented out?

Reply:
self-rekt :(